### PR TITLE
Add 63-refactor-geth-packages idea

### PR DESCRIPTION
**Pull request by adambabik**
This is supposed to replace issue #63.

cc @PombeirP

### CODE REVIEW

File: 63-refactor-geth-packages.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+Description:
+`common.NodeManager` interface and all its mocks have to be removed from status-go. This interface is too large and is not useful at all.
+
+### Iteration 5: `geth/common` package removal
+Goal Date: 2018-04-06
+
+Description:
+`get/common` package contains everything. In this iteration, the goal is to remove it complete or at least move most of the functions to appropriate packages.
+
+### Iteration 6: A PR in status-react
```
Comment (adriacidre) [inline 63-refactor-geth-packages.md]: I'd say simplifying `node.NodeManager` api could be in here.

**Comment by pilu** [comment]
@adambabik shouldn't that file be in the `/ideas` folder?

File: 63-refactor-geth-packages.md (hunk 1 above)
Comment (adambabik) [reply 63-refactor-geth-packages.md]: What do we want to simplify more in there? In the first iteration, we eliminated async interface from there.

Comment (adriacidre) [reply 63-refactor-geth-packages.md]: Things like:

- Do not name it NodeManager or Manager as its not managing multiple nodes
- Rename methods indicating there is only support for one node `StartNode` -> `Start`, `StopNode` -> `Stop`...
- Allow start/stop associated managers (jailManager and txQueueManager) and leverage StatusBackend of that logic.

Comment (adambabik) [reply 63-refactor-geth-packages.md]: It's fixed.

**Comment by oskarth** [comment]
Added 63 to index https://github.com/status-im/ideas

**Pull request merged**